auacm/app/modules/submission_manager/judge.py

```diff
@@ -65,9 +62,6 @@ def evaluate(submission, uploaded_file, time_limit=10):
     :param uploaded_file: the uploaded file
     :return: the status of the submission (one of the status constants above)
     """
-    directory = directory_for_submission(submission)
-    os.mkdir(directory)
-    uploaded_file.save(os.path.join(directory, uploaded_file.filename))
```

Comment:
I think. Not 100% sure. Looking into it. Judge tests are definitely broken though.